spec-0004/index.md

```diff
@@ -100,22 +100,26 @@ Complete examples of how projects implement this in their CI setup are linked in
 After someone creates an issue on https://github.com/scientific-python/upload-nightly-action
 requesting access to upload wheels a human/admin has to respond to that request.
```

Comment:
Maybe we should move this after the next paragraph. This part is linked to the project, not the org itself.